mitodl/ocw-hugo-themes
# Fix: Missing search icon by using SVG

# What are the relevant tickets?
Closes https://github.com/mitodl/ocw-hugo-themes/issues/1186

# Description (What does it do?)
This PR replaces `i` tag search material-icon by using SVG of the search icon. This fixes the missing icon issue because by using SVG, we do not have to use `url()` syntax inside `<style>` (Hugo's `relativeUrls` cause an issue for that)

# Screenshots (if appropriate):
Search icon: <img width="52" alt="image" src="https://github.com/mitodl/ocw-hugo-themes/assets/109785089/51542676-1bf3-4f40-8375-2bc9bfce5dc1">

# How can this be tested?
1. Checkout to this branch
2. `yarn build /to/ocw-content-rc/ocw-www/ /to/ocw-hugo-projects/ocw-www/config-offline.yaml`
3. Go to your local directory, and open up `ocw-content-rc/ocw-www/dist/page/2.html` in your browser.
4. Make sure the search icon is appearing and it works.
5. Then, scroll below, and go to page number 1. 
6. Make sure the search icon is appearing and it works. Make sure it's mobile responsive as well.
